### PR TITLE
i can't think up a name lol

### PR DESCRIPTION
**Pull request by sourTaste000**
image

### CODE REVIEW

**Comment by sourTaste000** [comment]
1 more pr/issue ez

File: index.js [hunk 1]
```diff
@@ -110,17 +110,17 @@ let i = 0;
 
 client.on('messageReactionAdd', async (reaction, user) => {
     if(reaction.emoji.toString() === "⭐" && !pinnedMessages.includes(reaction.message.content)) {
-            client.channels.cache.get('579741237377236992').send(`${user.username} voted for starboard`);
-            if (reaction.count === 3) {
+            client.channels.cache.get('735922655735054479').send(`${user.username} voted for starboard`);
```
Comment (dominikaaaa) [inline index.js:113]: no

File: index.js [hunk 2]
```diff
@@ -110,17 +110,17 @@ let i = 0;
...
+            if (reaction.count === 1) {
```
Comment (dominikaaaa) [inline index.js:114]: also no

**Comment by sourTaste000** [comment]
oh i forgot

**Comment by sourTaste000** [comment]
lmao

**Pull request merged**